js/data/util/categorize.js

```diff
@@ -15,17 +15,28 @@
  * == BSD2 LICENSE ==
  */
 
+ /* jshint esversion:6 */
+
 var _ = require('lodash');
+var { GLUCOSE_MM } = require('../../data/util/constants');
```

Comment:
`GLUCOSE_MMOL`??

Reply:
Maybe `MMOL_CONV_FACTOR` ?

Reply:
I agree that it's not a great constant name.  I ended up keeping it as it was, and using the same name in the `blip` repo as well, but I'll figure out a better one and change them over.

I think I'll use `MMOLL_CONVERSION_FACTOR`
That's the most accurate, if not a little long.

Reply:
Going with `MGDL_PER_MMOLL` as per the recommendation in the `blip` PR